Marlin/Marlin_main.cpp

```diff
@@ -4617,11 +4617,11 @@ void home_all_axes() { gcode_G28(true); }
 
         #if ENABLED(AUTO_BED_LEVELING_LINEAR)
 
-//        mean += measured_z;                                  // I believe this is unused code?
-//        eqnBVector[abl_probe_index] = measured_z;            // I believe this is unused code?
-//        eqnAMatrix[abl_probe_index + 0 * abl2] = xProbe;     // I believe this is unused code?
-//        eqnAMatrix[abl_probe_index + 1 * abl2] = yProbe;     // I believe this is unused code?
-//        eqnAMatrix[abl_probe_index + 2 * abl2] = 1;          // I believe this is unused code?
```

Comment:
Oh!   I see this code block happens with a 'Q' option.    So in that case, the fix is a little bit more involved.    We need to also call this for each point sampled in this block.
```cpp
             incremental_LSF(&lsf_results, xProbe, yProbe, measured_z);
```